Arpeggiator.cpp

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
+#include "Arpeggiator.h"
+Arpeggiator::Arpeggiator()
+{
+    arpInputVolts = 0.f; chordType = " Major9 "; arpModeControlValue = 2; arpMode = " "; isHigh = true; clockInputVolts = 0.f; voltageThreshold = 10.f;
+}
```

Comment:
use the member initializer list.
Don't initialize member variables inside constructor bodies.
- [ ] revise